client/tokenized-payment-request/transformers/wc-to-stripe.js

```diff
@@ -13,26 +13,37 @@ import { __ } from '@wordpress/i18n';
 export const transformCartDataForDisplayItems = ( cartData ) => {
 	const displayItems = cartData.items.map( ( item ) => ( {
 		amount: parseInt( item.prices.price, 10 ),
-		// TODO: should we also add variation attributes?
```

Comment:
Removed the TODO and added the variation attributes to the description, since it was ez enough